tests/test_photon_ops.py

```diff
@@ -554,10 +534,10 @@ def test_config_rubin_diffraction_without_field_rotation():
 def test_config_rubin_diffraction_optics():
     """Check the config interface to RubinDiffractionOptics."""
 
-    image_pos = galsim.PositionD(3076.4462608524213, 1566.4896702703757)
+    stamp_center = galsim.PositionD(3076.4462608524213, 1566.4896702703757)
```

Comment:
ditto here and a few other places I guess.

Reply:
Fixing throughout.